packages/legacy-events/ResponderTouchHistoryStore.js

```diff
@@ -95,13 +95,14 @@ function resetTouchRecord(touchRecord: TouchRecord, touch: Touch): void {
 function getTouchIdentifier({identifier}: Touch): number {
   invariant(identifier != null, 'Touch object is missing identifier.');
   if (__DEV__) {
-    warningWithoutStack(
-      identifier <= MAX_TOUCH_BANK,
-      'Touch identifier %s is greater than maximum supported %s which causes ' +
-        'performance issues backfilling array locations for all of the indices.',
-      identifier,
-      MAX_TOUCH_BANK,
-    );
+    if (!(identifier <= MAX_TOUCH_BANK)) {
```

Comment:
Same for these. `<=` needs to flip to `>`, `<` needs to flip to `>=`, `>` needs to be `<=`, and `>=` needs to become `<`.